### PR TITLE
simplify function MatchProvisioner

### PR DESCRIPTION
**Pull request by Abirdcfly**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go [hunk 1]
```diff
@@ -417,10 +417,7 @@ var ebsVolumeFilter = VolumeFilter{
 	},
 
 	MatchProvisioner: func(sc *storage.StorageClass) (relevant bool) {
```
Comment (denkensk) [inline pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go:419]:
```suggestion
	MatchProvisioner: func(sc *storage.StorageClass) bool {
```

**Review by denkensk — COMMENTED** [review]
you can also clean `relevant`

File: pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go (hunk 1 above)
Comment (Abirdcfly) [reply pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go]: Updated, thanks!

**Review by damemi — COMMENTED** [review]
/approve

**Comment by alculquicondor** [comment]
Mike beat me to it :smile:  Thanks for the contribution!

**Pull request merged**